server/cmd/mmctl/commands/import.go

```diff
@@ -240,10 +250,28 @@ func importUploadCmdF(c client.Client, command *cobra.Command, args []string) er
 func importProcessCmdF(c client.Client, command *cobra.Command, args []string) error {
 	importFile := args[0]
 
+	isLocal, _ := command.Flags().GetBool("local")
+	isLegacy, _ := command.Flags().GetBool("legacy")
+	// in local mode, we tell the server to directly read from this file.
+	if isLocal && !isLegacy {
+		if _, err := os.Stat(importFile); errors.Is(err, os.ErrNotExist) {
+			return fmt.Errorf("file %s doesn't exist. NOTE: If this file was uploaded to the server, please use the --legacy flag to revert to old behavior.", importFile)
```

Comment:
"uploaded to the server" could also imply something like scp'ing the file to the server too. Maybe we change it to something like this?

```suggestion
			return fmt.Errorf("file %s doesn't exist. NOTE: If this file was uploaded via `mmctl import upload`, please use the --legacy flag to revert to old behavior.", importFile)
```